arkeo/chain.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "arkeo",
+  "status": "live",
+  "website": "https://arkeo.network/",
+  "network_type": "mainnet",
+  "chain_type": "cosmos",
+  "pretty_name": "Arkeo",
+  "chain_id": "arkeo-main-v1",
+  "bech32_prefix": "arkeo",
```

Comment:
Recommended to register this prefix here
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1877

I applied for a register there